modules/multi-architecture-upgrade-mirrors.adoc

```diff
@@ -7,17 +7,4 @@
 
 = Upgrading a cluster with multi-architecture compute machines
 
-You must perform an explicit upgrade command to upgrade your existing cluster to a cluster that supports multi-architecture compute machines.
-
-.Prerequisites
-
-* You installed the OpenShift CLI (`oc`). 
-
-.Procedure
-* To manually upgrade your cluster, use the following command: 
-[source, terminal]
-+
-----
-$ oc adm upgrade --allow-explicit-upgrade --to-image <image-pullspec> <1>
-----
-<1> You can access the `image-pullspec` object from the link:https://mirror.openshift.com/pub/openshift-v4/multi/clients/ocp/latest[mixed-arch mirrors page] in the `release.txt` file.
\ No newline at end of file
+To upgrade your cluster with multi-architecture compute machines, you must use the `candidate-{product-version}` update channel. For more information, see "Understanding upgrade channels".
```

Comment:
we need to mention that there are two different things that can be done here
- migrate from a single arch to a multi-arch payload
- upgrade within the multi-arch payload


we need to mention that the user needs to change the channel first using `oc adm upgrade channel` and then if the operation is a migration, do an `oc adm upgrade --to-multi-arch`  and if it is an upgrade, do an `oc adm upgrade`. let me verify these steps and make sure they work as expected

Reply:
i just remembered that migration is only supported from 4.13+. i'll get back to you on this